lib/services/ios-project-service.ts

```diff
@@ -57,6 +57,7 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 			frameworkFilesExtensions: [".a", ".framework", ".bin"],
 			frameworkDirectoriesExtensions: [".framework"],
 			frameworkDirectoriesNames: ["Metadata", "metadataGenerator", "NativeScript", "internal"],
+			frameworkVersion: require(this.$projectData.projectFilePath).nativescript["tns-ios"].version,
```

Comment:
same concerns as above:
also consider using `this.$fs.readJson` instead of require, as the require is caching the results
one more thing - won't this fail in case `tns-ios` does not exist in nativescript key?